data/strings/sound.txt

```diff
@@ -426,9 +499,135 @@
     tr = çıkın.
     uk = З'їзд.
     vi = Lối thoát.
-    zh-Hans = 出口.
+    zh-Hans = 驶出
     zh-Hant = 出口.
 
+  [onto]
+    comment = Used in dist_direction_onto_street in position 3 unless take_exit_number or onto_exit_number are specified.
+    en = onto
+    ar = إلى
+    be = на
+    cs = na
+    da = ad
+    de = auf
+    el = στην
+    es = hacia
+    es-MX = hacia
+    eu =
+    fa = ﺐﻫ
+    fi = tielle
+    fr = sur
+    hi = पर
+    hr = na
+    hu = a-re
+    id = ke
+    it = sulla
+    ja = 
+    ko =
+    mr = वर
+    nb = inn på
+    nl = naar
+    pl = na
+    pt = para
+    pt-BR = para
+    ro = pe
+    ru = на
+    sk = na
+    sv = in på
+    sw = kwenye
+    th = เข้าสู่
+    tr = yönünde
+    uk = на
+    vi = vào
+    zh-Hans = 上
+    zh-Hant = 上
+
+  [onto_exit_number]
+    comment = TODO: currently unused, consider removing. Used in dist_direction_onto_street instead of "onto" when there's an exit number after. Use "." for none
+    en = .
+    ar = .
+    be = .
+    cs = .
+    da = .
+    de = .
+    el = .
+    es = .
+    es-MX = .
+    eu = .
+    fa = .
+    fi = .
+    fr = .
+    hi = .
+    hr = .
+    hu = .
+    id = .
+    it = .
+    ja = .
+    ko = .
+    mr = .
+    nb = .
+    nl = .
+    pl = .
+    pt = .
+    pt-BR = .
+    ro = .
+    ru = .
+    sk = .
+    sv = .
+    sw = .
+    th = .
+    tr = .
+    uk = .
+    vi = .
+    zh-Hans = .
+    zh-Hant = .
+
+  [take_exit_number]
+    comment = Used in dist_direction_onto_street instead of "Exit" when there's an exit number after. Replaces "onto" unless onto_exit_number is specified.
+    en = Take exit
+    ar = خروج إلى
+    be = З'езд на
+    ca = Agafa la sortida
+    cs = výstup na
+    da = Afkørsel ad
+    de = Nehmen Sie die Ausfahrt auf
+    el = Εξοδος στην
+    es = Tome la salida
+    es-MX = Tome la salida
+    eu = Irteera
+    fa = خروجﺐﻫ
+    fi = Poistu tielle
+    fr = Sortie sur
+    hi = बाहर जाएं पर
+    hr = Izlaz na
+    hu = Kilépés a-re
+    id = Keluar ke
+    it = Uscita sulla
+    ja = 出口を出る
+    ko = 출구
+    mr = बाहेर पडा वर
+    nb = Ta avkjøringen inn på
+    nl = Verlaat naar
+    pl = Wybierz zjazd
+    pt = Saia para
+    pt-BR = Pegue a saída para
+    ro = Ieșire pe
+    ru = Съезд на
+    sk = Výjazd na
+    sv = Utgång in på
+    sw = Kuondoka kwenye
+    th = ทางออก เข้าสู่
+    tr = çıkın yönünde
+    uk = З'їзд на
+    vi = Lối thoát vào
+    zh-Hans = 驶出 上
+    zh-Hant = 出口 上
+
+  [take_exit_number_street_verb]
+    comment = Optionally used inside dist_direction_onto_street in position 5 when take_exit_number needs to be split into two parts. (There is no take_exit_number_street)
+    ja = 
```

Comment:
Are empty strings really needed?

Reply:
I can check what happens when I take them out, I am somewhat concerned that the automatic fallback logic might put i.e. English words in a Japanese translation if English uses a word like "onto" and Japanese doesn't. The current logic is tested and intentional to say we tested a string in this location but want it explicitly blank.

Reply:
Then it should be explicitly mentioned in the comment above.

Reply:
Per other comment, I've tested and explicitly commented.